### PR TITLE
Proper .dme seperation

### PR DESCRIPTION
**Pull request by TheFakeElon**
## About The Pull Request

Gives firefly it's own .dme, this makes file tracking and conflict solving astronomically easier

## Changelog
:cl:
code: adds firefly dme
/:cl:

### CODE REVIEW

File: firefly.dme [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+// END_DEFINE_INCLUDE
+
+// BEGIN_BEESTATION_INCLUDE
+#include "beestation.dme"
+// END_BEESTATION_INCLUDE
+
+// BEGIN_INCLUDE
+
+// END_INCLUDE
```
Comment (TheFakeElon) [inline firefly.dme:25]: firefly files go here

File: firefly.dme [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#define FILE_DIR .
+// END_FILE_DIR
+
+// BEGIN_PREFERENCES
+#define DEBUG
+// END_PREFERENCES
+
+// BEGIN_DEFINE_INCLUDE
+
+// END_DEFINE_INCLUDE
```
Comment (TheFakeElon) [inline firefly.dme:17]: firefly defines go here

**Comment by francinum** [comment]
This feels like it's going to be actual pain to manage.

**Comment by MCterra10** [comment]
literally stolen from austation

**Comment by MCterra10** [comment]
I have copyright on this define inclusion system you know

**Pull request closed without merging**